kohese-portal/client/src/components/dashboard/project-dashboard/status-dashboard/state-bar-chart/state-bar-chart.component.html

```diff
@@ -85,7 +85,10 @@ <h3>
     The padding-bottom value below is calculated via the following formula:
       padding-bottom = width * (viewBox height / viewBox width)
   -->
-  <svg class=svg-chart #chart viewBox="0 0 100 50" preserveAspectRatio="xMidYMin slice"></svg>
+  <svg  class=svg-chart #chart id="chart"
+        (window:resize)="onResize($event)"
+        viewBox="0 0 100 50"
+        preserveAspectRatio="xMidYMin slice"></svg>
```

Comment:
Close tag should align with open tag, not be at the end of a multi-line open tag.